File: Graph/linkedList.py
class Node:
    def __init__(self, data):
        self.data = data
        self.next = None

class linkedList:
    def __init__(self):
        self.head = None
        

if __name__=='__main__':
    llist = linkedList()
    first = Node(1)
    second = Node(2)
    third = Node(3)
    fourth = Node(4)

    llist.head = first
    first.next = second
    second.next = third
    third.next = fourth

    curr = llist.head
    while(curr is not None):
        print(curr.data)
        curr = curr.next

File: Graph/linkedList.cpp
#include <bits/stdc++.h>
using namespace std;

class Node{
    public:
        int data;
        Node* next = NULL;
};


void printList(Node* head)
{
    Node* curr;
    curr = head;
    while(curr!=NULL)
    {
        cout << curr->data << endl;
        curr = curr->next;
    }
}

int main()
{
    Node* head = NULL;
    Node* second = NULL;
    Node* third = NULL;

    head = new Node();
    second = new Node();
    third = new Node();

    head->data = 1;
    second->data = 2;
    third->data = 3;

    head->next = second;
    second->next = third;
    printList(head);
    return 0;
}
